expression-test/expression_test_parser.cpp

```diff
@@ -407,12 +407,12 @@ Ignores parseExpressionIgnores() {
 std::optional<TestData> parseTestData(const filesystem::path& path) {
     TestData data;
     auto maybeJson = readJson(path.string());
-    if (!maybeJson.is<JSDocument>()) { // NOLINT
+    if (!std::holds_alternative<JSDocument>(maybeJson)) { // NOLINT
```

Comment:
I'm not sure what was linted here, if someone could help me with that one I'd happily check and maybe remove the NOLINT comment

I'll do some searching in the meanting to try to figure out what analyzer I should run to test this

Reply:
Could you try removing it? If the CI passes we can remove it.

I think Mapbox used some kind of linter in the past, but we are not (yet).